client/src/components/FileCard.jsx

```diff
@@ -0,0 +1,121 @@
+import './FileCard.css'
+
+import { Folder, DocumentTxt, DocumentImage, Play } from 'grommet-icons'
+
+const FileCard = ({ name, path, type, size }) => {
+  const splittedNameArray = name.split('.')
+  const fileExtension = splittedNameArray[splittedNameArray.length - 1]
+
+  const renderFileIcon = (fileExtension) => {
+    if (fileExtension === 'txt') {
+      return <DocumentTxt color='green' />
+    } else if (fileExtension === 'jpg') {
+      return <DocumentImage color='yellow' />
+    } else if (fileExtension === 'mp4') {
+      return <Play color='blue' />
+    } else {
+      return <DocumentTxt color='green' />
+    }
+  }
+
+  return (
+    <li>
+      {type === 'folder' ? (
+        <div className='file__info'>
+          <Folder color='magenta' size='medium' />
+          <a href={`/api/content?path=${path}`} className='file__name'>
+            {name}
+          </a>
+        </div>
+      ) : (
+        <div className='file__info'>
+          {renderFileIcon(fileExtension)}
+          <div className='file__description--container'>
+            <a href={`/api/content?path=${path}`} className='file__name'>
+              {name}
+            </a>
+            <p className='file__subtitle'>{size}</p>
+          </div>
+        </div>
+      )}
+    </li>
+  )
+}
+
+export default FileCard
+
+// name	"@root"
+// type	"folder"
+// path	"/"
+// items
+// 0
+// name	"one"
+// path	"/one"
+// type	"folder"
+// 1
+// name	"x - y"
+// path	"/x - y"
+// type	"folder"
+// 2
+// name	"x y"
+// path	"/x y"
+// type	"folder"
+// 3
+// name	"x'y"
+// path	"/x'y"
+// type	"folder"
+// 4
+// name	"привіт"
+// path	"/привіт"
+// type	"folder"
+// 5
+// name	"11184.jpg"
+// path	"/11184.jpg"
+// type	"file"
+// size	"96.4 kB"
+// 6
+// name	"Kilted Yoga  - The Jersey Edition.mp4"
+// path	"/Kilted Yoga  - The Jersey Edition.mp4"
+// type	"file"
+// size	"5.01 MB"
+// 7
+// name	"hello.txt"
+// path	"/hello.txt"
+// type	"file"
+// size	"1.1 kB"
+
+//   <h3>icons
+//     // if {type} items is ... file - use icon ..., another  - we use icon

```

Comment:
please, never leave commented code in your PRs
you can add comments explaining the business logic
but never keep unused commented rubbish code in files